### PR TITLE
Add MIT license

### PR DESCRIPTION
**Pull request by deepakunni3**
@debugpoint136 Ready for review.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 GitTut
```
Comment (deepakunni3) [inline LICENSE.md:3]: @debugpoint136 is the organization name okay?

Comment (debugpoint136) [reply LICENSE.md:3]: Yup looks good. 👍

Comment (debugpoint136) [inline LICENSE.md:3]: Can we change the license though, I am not ready to open source this yet

Comment (debugpoint136) [reply LICENSE.md:3]: @deepakunni3

**Comment by deepakunni3** [comment]
I can respond here too. This is more for discussion purposes that doesn't have much to do with any particular line of code but definitely within the context of a PR.

**Review by debugpoint136 — APPROVED** [review]

**Pull request merged**